Library/ViewModels/PledgePaymentMethodsViewModelTests.swift

```diff
@@ -38,19 +50,30 @@ final class PledgePaymentMethodsViewModelTests: TestCase {
     let userCreditCard = GraphUserCreditCard.amex
 
     withEnvironment(apiService: mockService, currentUser: User.template) {
-      self.reloadPaymentMethods.assertDidNotEmitValue()
+      self.reloadPaymentMethodsCards.assertDidNotEmitValue()
+      self.reloadPaymentMethodsAvailableCardTypes.assertDidNotEmitValue()
+      self.reloadPaymentMethodsProjectCountry.assertDidNotEmitValue()
 
       self.vm.inputs.configureWith((User.template, Project.template, false))
       self.vm.inputs.viewDidLoad()
 
       self.scheduler.run()
 
-      self.reloadPaymentMethods.assertValue(response.me.storedCards.nodes)
+      self.reloadPaymentMethodsCards.assertValue(response.me.storedCards.nodes)
+      self.reloadPaymentMethodsAvailableCardTypes.assertValues([
```

Comment:
It's a little difficult to follow these assertions when they're split up. Is it possible to do the assertions on the tuples? Something like this:
```
self.reloadPaymentMethodsCardsAndAvailable.assertValues([(GraphUserCreditCard.visa, true), (GraphUser CreditCard.masterCard, true)...etc])
```